ContentLock/Client/src/localizations/tr.ts

```diff
@@ -0,0 +1,29 @@
+export default {
+    "contentLockDashboard": {
+        "label": "?�erik Kilidi",
```

Comment:
Thanks @busrasengul the unusual characters here. Is it because my machine does not have certain characters installed on my machine?

![image](https://github.com/user-attachments/assets/065aec57-df88-450e-a1a7-3603e6d8c838)